src/sentry/static/sentry/app/views/usageStats/usageStatsOrg.tsx

```diff
@@ -0,0 +1,248 @@
+
+type State = {
+  orgStats: OrganizationUsageStats;
+} & AsyncComponent['state'];
+
+class UsageStatsOrganization extends AsyncComponent<Props, State> {
+  getEndpoints(): ReturnType<AsyncComponent['getEndpoints']> {
+    const {organization} = this.props;
+
+    // TODO(org-stats): Allow user to pick date range
+    const fourWeeksAgo = moment().subtract(31, 'days').unix();
+    const today = moment().unix();
+
+    return [
+      [
+        'orgStats',
+        `/organizations/${organization.slug}/stats_v2/`,
+        {
+          query: {
+            start: fourWeeksAgo,
+            end: today,
+            interval: '1d',
+          },
+        },
+      ],
+    ];
+  }
+
+  /**
+   * Ignore this hard-coded method. API response is being changed so this will
+   * be amended in a few days.
+   */
+  get formattedOrgStats(): {
+    stats: any[];
+    total: string;
+    accepted: string;
+    dropped: string;
+    filtered: string;
+  } {
+    const {dataCategory} = this.state;
+
+    const formatOptions = {
+      isAbbreviated: dataCategory !== DataCategory.ATTACHMENTS,
+      useUnitScaling: dataCategory === DataCategory.ATTACHMENTS,
+    };
+
+    return {
+      stats: [],
+      total: formatUsageWithUnits(0, dataCategory, formatOptions),
+      accepted: formatUsageWithUnits(0, dataCategory, formatOptions),
+      dropped: formatUsageWithUnits(0, dataCategory, formatOptions),
+      filtered: formatUsageWithUnits(0, dataCategory, formatOptions),
+    };
+  }
+
+  renderCards() {
+    const {dataCategory, dataCategoryName} = this.props;
+    const {total, accepted, dropped, filtered} = this.formattedOrgStats;
+
+    const cardData = [
+      {
+        title: tct('Total [dataCategory]', {dataCategory: dataCategoryName}),
+        value: total,
+      },
+      {
+        title: t('Accepted'),
+        value: accepted,
+      },
+      {
+        title: t('Filtered'),
+        description: tct(
+          'Filtered [dataCategory] were blocked due to your inbound data filter rules',
+          {dataCategory}
+        ),
+        value: filtered,
+      },
+      {
+        title: t('Dropped'),
+        description: tct(
+          'Dropped [dataCategory] were discarded due to rate-limits, quota limits, or spike protection',
+          {dataCategory}
+        ),
+        value: dropped,
+      },
+    ];
+
+    return (
+      <CardWrapper>
+        {cardData.map((c, i) => (
+          <StyledCard key={i}>
+            <HeaderTitle>
+              <OverflowEllipsis>{c.title}</OverflowEllipsis>
+              {c.description && (
+                <QuestionTooltip size="sm" position="top" title={c.description} />
+              )}
+            </HeaderTitle>
+            <CardContent>
+              <OverflowEllipsis>{c.value}</OverflowEllipsis>
+            </CardContent>
+          </StyledCard>
+        ))}
+      </CardWrapper>
+    );
+  }
+
+  renderChart(e?: Error) {
+    // TODO(leedongwei): Poke someone for a error-state design
```

Comment:
In other parts of the application charts display IconWarning when they can't load their data. See discover or release details for examples.